draft-ftbs-rats-msg-wrap.md

```diff
@@ -146,7 +151,22 @@ When using CBOR, the value field is serialized as a CBOR bytes string.
 CBOR Tags used as CMW are derived from CoAP Content Format values.
 If a CoAP Content Format exists for a RATS conceptual message, the
 TN() transform defined in {{Appendix B of RFC9277}} can be used to
-derive a CBOR tag in range \[1668546817, 1668612095\].
+derive a corresponding CBOR tag in range \[1668546817, 1668612095\].
+
+The RATS conceptual message is first serialized according to the Content
+Format associated with the tag and then encoded as a CBOR byte string,
+to which the tag is prepended.
+
+### Use of Pre-existing CBOR Tags
+
+If a CBOR tag has been registered in association with a certain RATS
+conceptual message independently of a CoAP Content Format (i.e., it is
+not obtained by applying the TN() transform), it can be readily used as
+an encapsulation without the extra processing described in {{cbor-tag}}.
+
+A consumer can always distinguish tags that have been derived via TN()
+from tags that are not, and therefore apply the right decapsulation on
+receive.
```

Comment:
Maybe a quick 'how' to be on the safest side?

Reply:
done in f5e20f2